[Pages] Fix broken links (#1777)
cloudflare/cloudflare-docs: products/pages/src/content/tutorials/build-a-blog-using-nuxt-and-sanity/index.md

$ yarn add @nuxtjs/sanity @sanity/client
```

> To configure the plugin in your Nuxt.js application, you'll need to provide some configuration details. The easiest way to do this it to copy the `sanity.json` folder from your studio into your application directory (though there are other methods, too: [check out the docs!](https://sanity.nuxtjs.org/getting-started/quick-start).

Author: ```suggestion
To configure the plugin in your Nuxt.js application, you will need to provide some configuration details. The easiest way to do this is to copy the `sanity.json` folder from your studio into your application directory (though there are other methods, too: [refer to the nuxt/sanity documentation.](https://sanity.nuxtjs.org/getting-started/quick-start).
```

> To configure the plugin in your Nuxt.js application, you will need to provide some configuration details. The easiest way to do this is to copy the `sanity.json` folder from your studio into your application directory (though there are other methods, too: [refer to the nuxt/sanity documentation.](https://sanity.nuxtjs.org/getting-started/quick-start).

Reviewer: ```suggestion
To configure the plugin in your Nuxt.js application, you will need to provide some configuration details. The easiest way to do this is to copy the `sanity.json` folder from your studio into your application directory (though there are other methods, too: [refer to the `@nuxt/sanity` documentation](https://sanity.nuxtjs.org/getting-started/quick-start).
```
